src/routes/teams/+page.svelte

```diff
@@ -0,0 +1,65 @@
+<script lang="ts">
+  import { TextAlignment } from '$lib/public/text-alignment/text-alignment';
+  import { acmAI, acmAlgo, acmDesign, acmDev } from '$lib/public/legacy/acm-paths';
+  import Spacing from '$lib/public/legacy/spacing.svelte';
+  import Block from '$lib/components/block/block.svelte';
+  import PathSection from './path-section.svelte';
+</script>
+
+<svelte:head>
+  <title>Teams | ACM at CSUF</title>
+</svelte:head>
+
+<Spacing --min="175px" --med="200px" --max="200px" />
+
+<Block>
+  <h1 slot="headline" class="size-xl">Meet the Teams</h1>
+  <p slot="text" class="size-md">
+    Teams are committees that specialize in specific fields in the tech industry. We’ve designed the
+    teams to be gateways for students to explore new fields, develop new interests, and learn new
+    skills that will benefit them in the industry.
+  </p>
+</Block>
+
+<Spacing --min="100px" --med="175px" --max="200px" />
+
+<PathSection info={acmAI} textAlign={TextAlignment.RIGHT}>
+  <p slot="content" class="size-md">
+    This team is dedicated to providing accessible information about artificial intelligence and
+    machine learning to all. <span class="brand-emerald brand-em">AI</span> focuses on fun projects geared
```

Comment:
This is just a small nit. Definitely not necessary but it looks cleaner if we change it to this:
`<b class="acm-emerald">AI</b>`

Reply:
I like this idea, but it would needt o be consistent with the rest of the codebase. I think this should be discussed in a new issue.